packages/theme/src/Components/Radio/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { GeneratedIconProps, Icons } from '@masknet/icons'
+import type { FC } from 'react'
+
+export interface RadioProps extends GeneratedIconProps {
+    checked?: boolean
+}
+
+export const Radio: FC<RadioProps> = ({ checked, ...rest }) => {
```

Comment:
Please consider using https://react-spectrum.adobe.com/react-aria/useRadioGroup.html for better accessibility. e.g. Being able to handle keyboard events.

Reply:
The Radio style has been overwritten in the UI kit and can be used.